### PR TITLE
Jsummer/add looker support

### PR DESCRIPTION
**Pull request by sfc-gh-jsummer**
Redesign partner support for DBT and add support for Looker Explore semantic.

### CODE REVIEW

File: admin_apps/shared_utils.py [hunk 1]
```diff
@@ -829,261 +878,96 @@ def download_yaml(file_name: str, conn: SnowflakeConnection) -> str:
...
-        return obj.get("expr", obj["name"]).lower()
-
-    d = {}
-    for i in section_names:
-        d[i] = {extract_key(obj): obj for obj in data_dict.get(i, [])}
+    query_tag = get_sit_query_tag(vendor, action)
 
-    return d
+    conn.cursor().execute(f"alter session set query_tag='{query_tag}'")
+    conn.cursor().execute("SELECT 'SKIMANTICS';")
```
Comment (sfc-gh-cnivera) [inline admin_apps/shared_utils.py:911]: Is this meant to be hardcoded?

File: admin_apps/shared_utils.py [hunk 2]
```diff
@@ -829,261 +878,96 @@ def download_yaml(file_name: str, conn: SnowflakeConnection) -> str:
...
-    Loads one or more YAML files and combines them into a single list.
-    """
-    combined_yaml = []
-    for file_path in file_paths:
-        yaml_content = yaml.safe_load(file_path)
-        combined_yaml.append(unpack_yaml(yaml_content))
-    return combined_yaml
+    query_tag = {
+        "origin": "sf_sit",
+        "name": "skimantics",
```
Comment (sfc-gh-cnivera) [inline admin_apps/shared_utils.py:891]: Is this query tag meant to be hardcoded?

File: admin_apps/journeys/builder.py [hunk 3]
```diff
@@ -133,6 +95,7 @@ def table_selector_dialog() -> None:
         placeholder="Select the databases that contain the tables you'd like to include in your semantic model.",
         on_change=update_schemas_and_tables,
         key="selected_databases",
+        # default=st.session_state.get("selected_databases", []),
```
[the remark this answers is not in the document]
Comment (sfc-gh-jsummer) [reply admin_apps/journeys/builder.py:98]: I was going to say that was a mistake from prior experimentation and be kept. However, uncommenting the line and running the streamlit app renders this warning in the app: 
`The widget with key "selected_databases" was created with a default value but also had its value set via the Session State API.`
So I think we should remove it unless you see any downstream implications.

File: admin_apps/shared_utils.py (hunk 1 above)
Comment (sfc-gh-jsummer) [reply admin_apps/shared_utils.py:911]: Yes, it is. I'm open to alternative implementations of course. Functions set_sit_query_tag and get_sit_query_tag are utility functions explicitly to add Solution Innovation Tag (SIT) query tags that the team uses to track solution usage and consumption. 

`SELECT SKIMANTICS` is just a zero-consumption query on which we will set the query tag that we will use to track customers proceeding through the partner support funnel.

File: admin_apps/shared_utils.py (hunk 2 above)
Comment (sfc-gh-jsummer) [reply admin_apps/shared_utils.py:891]: Yes, it is. I'm open to alternative implementations of course. Functions set_sit_query_tag and get_sit_query_tag are utility functions explicitly to add Solution Innovation Tag (SIT) query tags that the team uses to track solution usage and consumption. 

The official tag for SIT to track this particular effort is hard-coded in the highlighted function.

File: admin_apps/journeys/builder.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (sfc-gh-cnivera) [reply admin_apps/journeys/builder.py:98]: sgtm! I think it still works as intended when being commented out. If it is indeed being set somewhere else then I don't think we need this line anymore.

File: admin_apps/shared_utils.py (hunk 1 above)
Comment (sfc-gh-cnivera) [reply admin_apps/shared_utils.py:911]: Gotcha, I wasn't familiar with this and the query tag, but that context makes sense. Just wanted to double check it wasn't being left in for testing purposes.

**Review by sfc-gh-cnivera — APPROVED** [review]

**Comment by sfc-gh-cnivera** [comment]
Thanks! Will run through Looker flows once I have creds set up. Mainly ran through builder + iterator flows to ensure existing functionality.

**Pull request merged**